portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 		WebContent.viewPGViaWCD(webContentContent = "WC WebContent Content");
 	}
 
+	@description = "This is a test for LPS-129438. View the changes of Border Styles in WCD on widget page."
+	@priority = "4"
+	test ConfigureLookAndFeelBorderStyleOfWCD {
+		task ("Assert the default value in border style on WCD") {
+			Navigator.gotoSitePage(
+				pageName = "Web Content Display Page",
+				siteName = "Test Site Name");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
+				value1 = "rgba(0, 0, 0, 0)");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-width",
+				value1 = "1px");
```

Comment:
View border-top-width, border-left-width, border-right-width and border-bottom-width